guides/spin/cli/index.md

```diff
@@ -90,14 +92,47 @@ auth:
 Read [the OAuth setup instructions](https://www.spinnaker.io/setup/security/authentication/oauth/providers/)
 to see examples for acquiring a clientId/clientSecret from your provider.
 
-Unlike X.509, OAuth2 needs to be initialized once to authenticate with the provider before
+This OAuth2 configuration method needs to be initialized once to authenticate with the provider before
 it can be used for automation. To authenticate, configure OAuth2 as shown above and execute
 any `spin` command. You will be prompted to authenticate with your OAuth2 provider
 and paste an access code. `spin` then exchanges the code for an OAuth2 access/refresh token pair,
 which it caches in your `~/.spin/config` file for future use. All subsequent `spin` calls will
 use the cached OAuth2 token for authentication with no user input required. If an OAuth2
 access token expires, `spin` will use the refresh token to renew the access token expiry.
 
+#### Access and Refresh Token
+
+`spin` can also be configured with an OAuth2 access token and refresh token in lieu of configuring
+an OAuth2 client ID and secret. The `spin` configuration looks like this:
+
+```yaml
+auth:
+  enabled: true
+  oauth2:
+    authUrl: # OAuth2 provider auth url
+    tokenUrl: # OAuth2 provider token url
+    # no clientId or clientSecret
+    scopes: # Scopes requested for the token
+    - scope1
+    - scope2
+    cachedToken:
+      accesstoken: ${ACCESS_TOKEN} # Note the key capitalization
+      refreshtoken: ${REFRESH_TOKEN} # Note the key capitalization
+```
+
+This method is OAuth2-provider specific since the workflow to acquire
+a token is different for each provider.
+
+* Google OAuth2 with `gcloud`
```

Comment:
One bullet by itself is kinda lonely. Maybe make this a second sentence in the prev. paragraph? 
"To do so for Google OAuth2:
1. Authenticate via `gcloud auth login`
2. Use the following commands to acquire the tokens:
  - (commands)
"

Reply:
The idea is that we'd have a list for all the supported providers. Ideally this would be its own header, but markdown headers don't nest 5 levels deep unfortunately. Should I still roll this into the previous paragraph?

Reply:
What do you think is the timeframe for having n+1?

Reply:
I don't know, we'll address this when we have to.